### ISSUE TITLE
[Feat] jupyter: Support custom CRS with Proj4Leaflet

### ISSUE THREAD

**Issue by ninsbl** [OSGeo/grass#3446]
**Is your feature request related to a problem? Please describe.**
User data is regularly in other CRS than WGS84 or WebMercator. It would be great if grass.jupyter integration could handle custom CRS to avoid reprojection. I remember there was a discussion earlier, but it seams leaflet now has better supprt for other CRS with the [proj4leaflet plugin](https://github.com/kartena/Proj4Leaflet), see example here:
https://github.com/jupyter-widgets/ipyleaflet/blob/master/examples/CustomProjections.ipynb

**Describe the solution you'd like**
Maps in jupyter should be able to set the CRS according to the location / Project instead of allways reprojecting in order to avoid costly computations (however, I did not check (or clearly understand) what proj4leaflet actually does and if it really avoids reprojecting)...
It would add to dependencies though...

**Comment by petrasovaa** [comment]
I agree this should be explored. But the limitation will be the basemaps that are in pseudomercator only...
